src/main/kotlin/lotto/model/Lotto.kt

```diff
@@ -0,0 +1,44 @@
+package lotto.model
+
+interface Lotto {
+    fun matchCount(winningNumbers: List<Int>): Int
```

Comment:
As I mentioned [here](https://github.com/herotechcourse/kotlin-lotto/pull/16#issuecomment-3045758986), it’s better to stick with a regular `class` for now.